assets/wire/components/filters/FilterGroup.tsx

```diff
@@ -70,16 +70,17 @@ export default function FilterGroup({group, activeFilter, aggregations, toggleFi
     const activeBuckets: Array<any> = (get(activeFilter, group.field) || [])
         .map((filter: any) => ({key: filter}));
     const bucketPath: string = get(group, 'agg_path') || `${group.field}.buckets`;
-    const buckets = uniqBy(
+    const allBuckets = uniqBy(
         cloneDeep(get(aggregations, bucketPath) || group.buckets || []).concat(activeBuckets) as Array<IBucket>,
         'key',
-    )
+    );
+    const bucketsBySearchTerm = allBuckets
         .sort(compareFunction)
         .filter(({key}: any) => searchTerm.length > 0 ? key.toString().toLocaleLowerCase().includes(searchTerm) : true);
 
     return (
         <NavGroup key={group.field} label={group.label}>
-            {buckets.length > 50 && (
+            {allBuckets.length > 50 && (
```

Comment:
why `50`? shouldn't you use a `pageSize` variable here?

Reply:
50 was in the ticket requirement